CineElorrietaMVC/src/main/java/Modelo/Tickets.java

```diff
@@ -0,0 +1,84 @@
+package Modelo;
+
+import java.util.Date;
+
+public class Tickets{
+	private int NumTrans;
+	private Date fecha = new Date();
+	private String NomLocal;
+	private ProductosElegidos [] productos = new ProductosElegidos[20];
```

Comment:
Usad un array dinamico (arraylist) o id redimensionando el array según añadáis productos

Reply:
Solucionado